src/app/embed/adapters/draper.adapter.ts

```diff
@@ -39,18 +39,24 @@ export class DraperAdapter extends FeedAdapter {
     props = super.processDoc(doc, props);
     props.feedArtworkUrl = this.getTagAttributeNS(doc, 'rp', 'image', 'href')
                         || props.feedArtworkUrl;
-    props.subscribeUrl = `https://play.radiopublic.com/${this.getTagTextNS(doc, 'rp', 'program-id')}`;
+    props.programId = this.getTagTextNS(doc, 'rp', 'program-id');
```

Comment:
Is this used anywhere besides the `subscribeUrl`?  Just storing this for posterity?